### PR TITLE
Add inspect step to perform introspection

### PR DESCRIPTION
**Pull request by bfournie**
This will run introspection on a manageable node. We should be able to remove this step when using fast-track deployment but its useful for now to gather introspection data.

To get introspection data:
GET http://<IP>:5050/v1/introspection/<node>/data

or via openstackclient:
export OS_TOKEN=fake-token
export OS_URL=http://localhost:6385/
openstack baremetal introspection data save <node>

### CODE REVIEW

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/139/

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/140/

File: 07_deploy_masters.sh [hunk 1]
```diff
@@ -37,6 +37,7 @@ for node in $(jq -r .nodes[].name $MASTER_NODES_FILE); do
   # FIXME(shardy) we should parameterize the image
   openstack baremetal node set $node --instance-info "image_source=http://172.22.0.1/images/$RHCOS_IMAGE_FILENAME_LATEST" --instance-info image_checksum=$(curl http://172.22.0.1/images/$RHCOS_IMAGE_FILENAME_LATEST.md5sum) --instance-info root_gb=25 --property root_device="{\"name\": \"$ROOT_DISK\"}"
   openstack baremetal node manage $node --wait
+  openstack baremetal node inspect $node --wait
```
Comment (derekhiggins) [inline 07_deploy_masters.sh]: In my local env this adds over 8 minutes to the runtime for this script, could you do them in parallel similar to how we do the deploy so it should only add 3 minutes or so?

**Comment by derekhiggins** [comment]
see comment inline, also a small typo in the commit mesage

> export OS_URL=http://localhost:6385/

that line from the commit message should be :5050 I think?

File: 07_deploy_masters.sh (hunk 1 above)
Comment (dtantsur) [reply 07_deploy_masters.sh]: Yeah, waiting for each node is a waste of time. We probably need to start inspection for each, then loop waiting for "manageable" again.

**Comment by derekhiggins** [comment]
The commit message has the same info twice, can you remove part of it. LGTM, assuming CI is happy

File: 07_deploy_masters.sh (hunk 1 above)
Comment (bfournie) [reply 07_deploy_masters.sh]: Fixed to add loop to check for manageable.

**Comment by bfournie** [comment]
> The commit message has the same info twice, can you remove part of it. LGTM, assuming CI is happy

Thanks, fixed.

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/155/

**Pull request merged**